DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -1251,6 +1251,8 @@ extension Tab/*: NavigationResponder*/ { // to be moved to Tab+Navigation.swift
 
     @MainActor
     func webContentProcessDidTerminate(with reason: WKProcessTerminationReason?) {
+        guard (error?.code.rawValue ?? WKError.Code.unknown.rawValue) != WKError.Code.webContentProcessTerminated.rawValue else { return }
```

Comment:
try to avoid duplicate executions of this in case the app is stuck in some kind of loop